---

input_file: "pcd_images/shape3/shape3_60deg.pcd"
output_file: "pcd_images/shape3/shape3_60deg_test.pcd"
save_output: true
translate_output: false
automatic_bounds: false

filter_box: [-1, 1, -1, 1, 0.005, 1] # bounding box limits
#filter_box: [-0.3, 0.0, -0.8, -0.5, 0.005, 0.25] # bounding box limits

voxel_leaf_size: 0.001                          # voxel leaf size
